mappings/net/minecraft/util/IntHashMap.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS none/om net/minecraft/util/IntHashMap
+	CLASS none/om$a Element
+		FIELD a key I
+		FIELD b value Ljava/lang/Object;
+		FIELD c next Lnone/om$a;
+		FIELD d hash I
+		METHOD <init> (IILjava/lang/Object;Lnone/om$a;)V
+			ARG 0 hash
+			ARG 1 key
+			ARG 2 value
+			ARG 3 next
+		METHOD a getKey ()I
+		METHOD b getValue ()Ljava/lang/Object;
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 0 obj
+	FIELD a elements [Lnone/om$a;
+	FIELD b count I
+	FIELD c capacity I
+	FIELD d capacityFactor F
+	METHOD a get (I)Ljava/lang/Object;
+		ARG 0 key
+	METHOD a getMinElementIndex (II)I
+		ARG 0 hash
+		ARG 1 length
+	METHOD a addInternal (IILjava/lang/Object;I)V
```

Comment:
Actually, I named it `addInternal` because it's in charge of adding a **new** element, `put` also replaces existing elements.